package.json

```
@@ -10,7 +10,8 @@
     "precommit": "lint-staged"
   },
   "lint-staged": {
-    "*.{js,json,css,md}": ["prettier --write", "git add"]
+    "*.{css,md,json}": ["prettier --write", "git add"],
+    "*.{js}": ["./node_modules/.bin/eslint --fix", "git add"]
```

no es necesario agregar la ruta completa a eslint, basta con usar eslint simplemente
```suggestion
    "*.{js}": ["eslint --fix", "git add"]
```

---

@madkoding estoy 99% seguro que usando la forma que te índico primero busca de forma local eslint y liego de forma global

---

lo probe, funciona bien como dices